frontend/src/container/ListOfDashboard/utils.ts

```diff
@@ -3,18 +3,18 @@ import { Dashboard } from 'types/api/dashboard/getAll';
 export const filterDashboard = (
 	searchValue: string,
 	dashboardList: Dashboard[],
-): any[] => {
+): Dashboard[] => {
 	// Convert the searchValue to lowercase for case-insensitive search
 	const searchValueLowerCase = searchValue.toLowerCase();
 
 	// Use the filter method to find matching objects
 	return dashboardList.filter((item: Dashboard) => {
 		// Convert each property value to lowercase for case-insensitive search
 		const itemValues = Object.values(item?.data).map((value) =>
-			value.toString().toLowerCase(),
+			value?.toString().toLowerCase(),
```

Comment:
can we filter out the null values and then operate on the remaining data? protects us from adding null checks everywhere

Reply:
Done.